### PR TITLE
Andrey gvozdev/scaffolding

### PR DESCRIPTION
**Pull request by Andrey-Gvozdev**

### CODE REVIEW

File: DIMS-Core.Database/DIMS-Core.Database/Views/vUserTask.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE VIEW [dbo].[vUserTask]
+	AS SELECT
+	UserTask.UserId AS UserID,
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Views/vUserTask.sql:3]: Don't use 'ID' at the end please. Just 'Id'.

**Review by ShlemenKirill — APPROVED** [review]

File: DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs [hunk 2]
```diff
@@ -15,18 +15,26 @@ public DIMSCoreContext(DbContextOptions<DIMSCoreContext> options)
...
+        public virtual DbSet<Task> Task { get; set; }
+        public virtual DbSet<TaskState> TaskState { get; set; }
+        public virtual DbSet<TaskTrack> TaskTrack { get; set; }
+        public virtual DbSet<UserProfile> UserProfile { get; set; }
+        public virtual DbSet<UserTask> UserTask { get; set; }
+        public virtual DbSet<VTask> VTask { get; set; }
+        public virtual DbSet<VUserProfile> VUserProfile { get; set; }
+        public virtual DbSet<VUserProgress> VUserProgress { get; set; }
+        public virtual DbSet<VUserTask> VUserTask { get; set; }
+        public virtual DbSet<VUserTrack> VUserTrack { get; set; }
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs:28]: All these props must be in plural form.

File: DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs [hunk 3]
```diff
@@ -15,18 +15,26 @@ public DIMSCoreContext(DbContextOptions<DIMSCoreContext> options)
...
-//            }
-//        }
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("Data Source=(localdb)\\MSSQLLocalDB;Database=DIMS-Core;Trusted_Connection=True;");
+            }
+        }
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs:37]: This setting exists in database json. Remove it.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
I did some comments. Fix them and let me know

**Review by PushMeNow — APPROVED** [review]
Approve

**Pull request merged**